spec/factories/collections.rb

```diff
@@ -152,56 +141,26 @@
   #                      :allow_multiple_membership, # OR :not_allow_multiple_membership
   #                    ] }
   #   let(:collection_type) { create(:collection_lw_type, settings) }
-  #   let(:collection) { build(:collection_lw, collection_type_gid: collection_type.gid) }
```

Comment:
Hmm…this documentation looks like it might be nice to have.

Reply:
Oh, I removed it by overwriting with current factory code, but missed copying the current documentation. I'll add that back in.

Reply:
which documentation? This portion of the code was basically a copy of Hyrax's so I copied over their current documentation with their current factory code.

Reply:
Basically, the nesting logic and documentation was removed.